apps/condo/domains/common/components/TableFilter.tsx

```diff
@@ -21,13 +21,15 @@ export interface IFilterContainerProps {
 
 const FILTER_CONTAINER_STYLES: CSSProperties = { padding: 16 }
 
+const handleStopPropagation = (e) => e.stopPropagation()
+
 export const FilterContainer: React.FC<IFilterContainerProps> = (props) => {
     const { showClearButton, clearFilters, children } = props
     const intl = useIntl()
     const ResetLabel = intl.formatMessage({ id: 'filters.Reset' })
 
     return (
-        <div style={FILTER_CONTAINER_STYLES}>
+        <div style={FILTER_CONTAINER_STYLES} onKeyDown={handleStopPropagation}>
```

Comment:
Does this fix actual for "callcenter" app only.
Is it presented in "condo" app?

Reply:
it was fixed in all tables, who use these containers. And "condo", and "callcenter"